src/GraphQL/Instrumentation/FieldMiddlewareBuilder.cs

```diff
@@ -9,17 +9,17 @@ namespace GraphQL.Instrumentation
 {
     public interface IFieldMiddlewareBuilder
     {
-        IFieldMiddlewareBuilder Use(Func<FieldMiddlewareDelegate, FieldMiddlewareDelegate> middleware);
+        IFieldMiddlewareBuilder Use(Func<ISchema, FieldMiddlewareDelegate, FieldMiddlewareDelegate> middleware);
```

Comment:
@Shane32 in order to achieve the desired behavior, we will have to change the interface to pass schema down. There is an extension method that has the old signature so clients need only to recompile their code:
```c#
public static IFieldMiddlewareBuilder Use(this IFieldMiddlewareBuilder builder, Func<FieldMiddlewareDelegate, FieldMiddlewareDelegate> middleware)
            => builder.Use((_, next) => middleware(next));
```